aleph-client/src/session.rs

```diff
@@ -86,31 +86,56 @@ pub fn set_keys(connection: &SignedConnection, new_keys: Keys, status: XtStatus)
 
 /// Get the number of the current session.
 pub fn get_current<C: AnyConnection>(connection: &C) -> u32 {
+    get_session(connection, None)
+}
+
+pub fn get_session<C: AnyConnection>(connection: &C, block_hash: Option<H256>) -> u32 {
```

Comment:
Changed. Is there any other additional value of such aliases beyond just their more verbose name - any type level/correctness related value (I can still use u32, ugly `as` operator and so on...).

Reply:
I do not see other advantages, but verbosity is quite enough for me. I wouldn't have much against `struct SessionIndex(u32)` but this would require `.0` in many places